src/Modules/SimplCommerce.Module.Reviews/Controllers/ReviewController.cs

```diff
@@ -1,20 +1,27 @@
-﻿using System.Threading.Tasks;
-using Microsoft.AspNetCore.Mvc;
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
 using SimplCommerce.Infrastructure.Data;
+using SimplCommerce.Module.Catalog.Models;
 using SimplCommerce.Module.Core.Extensions;
 using SimplCommerce.Module.Reviews.Models;
 using SimplCommerce.Module.Reviews.ViewModels;
+using System.Linq;
+using System.Threading.Tasks;
 
 namespace SimplCommerce.Module.Reviews.Controllers
 {
     public class ReviewController : Controller
     {
+        private const int DefaultPageSize = 25;
+
         private readonly IRepository<Review> _reviewRepository;
+        private readonly IRepository<Product> _productRepository;
```

Comment:
The review module should not depend on the Product, so that we might use this review modules for others such as vendors, articles